commune_app/tests/test_commune.py

```diff
@@ -38,3 +32,11 @@ def test_create_delete_commune(self):
         new_commune.delete()
         with pytest.raises(Exception):
             Commune.objects.get(id=new_commune.id)
+
+    def test_wallet(self, commune0):
+        commune0.wallet_charge(-1000)
```

Comment:
I think instead of this line it will be better to initiate the commune0 with wallet=1000